bin/trigger

```diff
@@ -1,18 +1,23 @@
 #!/bin/bash
 set -euo pipefail
 
+TOKEN="${PAT_GITHUB_DISPATCH:-}"
```

Comment:
You can do this to preserve the original env var name but ensure it's a declared var for `set -u`.

```suggestion
: "${PAT_GITHUB_DISPATCH:=}"
```

Using the original env var name in the code rather than a new name makes it more clear where the env var is used when grepping.

Reply:
Thanks for reviewing! Updated in [46962f8](https://github.com/nextstrain/ncov-ingest/pull/299/commits/46962f8d0f37d0b4e93ab03811dfd61bf3056e7c)